sklbench/benchmarks/sklearn_estimator.py

```diff
@@ -334,7 +334,9 @@ def verify_patching(stream: io.StringIO, function_name) -> bool:
     return acceleration_lines > 0 and fallback_lines == 0
 
 
-def create_online_function(method_instance, data_args, batch_size):
+def create_online_function(
+    estimator_instance, method_instance, data_args, num_batches, batch_size

```

Comment:
Where `num_batches` is used in this function?

Reply:
fixed